src/components/ErrorBoundary/ErrorBoundary.test.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
```

Comment:
If we're no longer using `withErrorBoundary`, is there a need for `ErrorBoundary` related components/tests? Aka can we kill more things with 🔥